### PR TITLE
:sparkles: Fixes #216 feature added to enable/disable default child g…

### PR DESCRIPTION
**Pull request by vatsaltanna**
…estures of showcase view

### CODE REVIEW

File: example/lib/main.dart [hunk 1]
```diff
@@ -181,9 +181,14 @@ class _MailPageState extends State<MailPage> {
                                     Showcase(
                                       key: _one,
                                       description: 'Tap to see menu options',
-                                      child: Icon(
-                                        Icons.menu,
-                                        color: Theme.of(context).primaryColor,
+                                      disableDefaultChildGestures: true,
+                                      child: GestureDetector(
+                                        onTap: () =>
+                                            print('menu button clicked'),
```
Comment (DhavalRKansara) [inline example/lib/main.dart:187]: use debugPrint here.

File: lib/src/showcase.dart [hunk 2]
```diff
@@ -100,11 +101,20 @@ class Showcase extends StatefulWidget {
...
     this.onTargetDoubleTap,
     this.tipBorderRadius,
+    this.disableDefaultChildGestures = false,
   })  : height = null,
         width = null,
         container = null,
         assert(overlayOpacity >= 0.0 && overlayOpacity <= 1.0,
             "overlay opacity must be between 0 and 1."),
+        assert(
+            !(disableDefaultChildGestures &&
```
Comment (DhavalRKansara) [inline lib/src/showcase.dart:111]: we can remove this assertion.

File: lib/src/showcase.dart [hunk 3]
```diff
@@ -144,8 +154,17 @@ class Showcase extends StatefulWidget {
     this.onTargetLongPress,
     this.onTargetDoubleTap,
     this.tipBorderRadius,
+    this.disableDefaultChildGestures = false,
   })  : showArrow = false,
         onToolTipClick = null,
+        assert(
+            !(disableDefaultChildGestures &&
```
Comment (DhavalRKansara) [inline lib/src/showcase.dart:161]: we can remove this assertion.

File: example/lib/main.dart (hunk 1 above)
Comment (DhavalRKansara) [inline example/lib/main.dart:187]: Use debugPrint

**Review by DhavalRKansara — CHANGES_REQUESTED** [review]
@vatsaltanna  Few Comments added

File: lib/src/showcase.dart (hunk 2 above)
Comment (vatsaltanna) [reply lib/src/showcase.dart:111]: reason ?

**Review by DhavalRKansara — APPROVED** [review]

**Pull request merged**